R/bootstrap-model.R

```diff
@@ -448,8 +448,33 @@ summarize_bootstrap_run <- function(
       )
     }
 
-    boot_spec <- get_boot_spec(.boot_run)
+    # Update spec to store bbi_version and configuration details
+    #  - so functions like config_log have to do less of a lift
+    boot_models <- get_boot_models(.boot_run)
+
+    # These should be consistent across all models
+    config_lst <- purrr::map(boot_models, function(.m){
+      path <- get_config_path(.m, .check_exists = FALSE)
+      config <- jsonlite::fromJSON(path)
+      # dont store timeout - unique to the individual model
+      config$configuration$parallel_timeout <- NULL
+      list(bbi_version = config$bbi_version, configuration = config$configuration)
+    }) %>% unique()
```

Comment:
I actually had this exact conversation with @seth127 and brought up the same point (despite implementing it this way). Short answer is I wasn't sure. 

I will say this operation is remarkably quick, and in fact a similar operation is performed every time you print a bootstrap run to the console, until it has been "cleaned up" (as in it iterates through all the models to determine their status, to determine the overall status).

I cant imagine how these values would differ unless you were running `bbi` directly (in which case a lot of the necessary `bbr` setup wouldn't even happen), but we thought it wouldn't hurt to include given that it didn't noticeably impact performance.

Reply:
Yes, as @barrettk noted, we discussed this briefly. Given that it's a very quick operation (just checking the existence of the `bbi_config.json`), I lean toward doing the full uniqueness check. Especially since this is called inside `summarize_bootstrap()` which will either 1) have other elements that take significantly longer or 2) already be cached.

If @kyleam feels strongly that this is wasteful, I'm open to reconsidering, but I think this is the safer approach and doesn't seem to add much practical overhead.

Reply:
I don't consider it a blocker, but I'm not in favor of it.  Unless we can name what we're guarding against, it's not worth the code in my opinion.  There are endless ways a user could corrupt things by mucking around on disk.